docs/src/main/paradox/directives/linking.md

```diff
@@ -97,7 +97,14 @@ For example, given:
  - `javadoc.akka.http.base_url=http://doc.akka.io/japi/akka-http/10.0.0`
 
 Then `@javadoc[Http](akka.http.javadsl.Http#shutdownAllConnectionPools--)` will resolve to
-<http://doc.akka.io/japi/akka-http/10.0.0/akka/http/javadsl/Http.html#shutdownAllConnectionPools-->.
+<http://doc.akka.io/japi/akka-http/10.0.0/?akka/http/javadsl/Http.html#shutdownAllConnectionPools-->.
+
+The `@javadoc` directive offers two styles of linking, linking to the frames version of
+the apidocs, and linking to the non frames version. This can be controlled using the `javadoc.link_style`
+property, setting it to `frames` for frames style linking, and `no-frames` for no frames style linking.
```

Comment:
I wonder if `no-frames` is the best we can come up with. Maybe `page` would work? But this implementation accepts everything that is not `frames`, right?

Reply:
Correct. I'll change to page.

Reply:
I updated to `direct`, because it links direct to the page, instead of through the frames.